src/libsyntax/feature_gate.rs

```diff
@@ -409,6 +409,12 @@ impl<'a> Context<'a> {
 
 pub fn emit_feature_err(diag: &SpanHandler, feature: &str, span: Span, explain: &str) {
     diag.span_err(span, explain);
+
+    // #23973: do not suggest `#![feature(...)]` if we are in beta/stable
```

Comment:
(Oh, don't get me wrong; I'm a heavy user of `blame` too, via `M-x vc-annotate`.  But that can require tracing through a chain of history as code gets moved around or inconsequentially revised by intermediate authors.)